deltachat-ios/Controller/AccountSetup/SecuritySettingsController.swift

```diff
@@ -1,24 +1,16 @@
 import UIKit
 import DcCore
 
+protocol SecuritySettingsDelegate: AnyObject {
+    func onSecuritySettingsChanged(newValue: Int)
+}
+
 class SecuritySettingsController: UITableViewController {
 
     private var options: [Int32] = [DC_SOCKET_AUTO, DC_SOCKET_SSL, DC_SOCKET_STARTTLS, DC_SOCKET_PLAIN]
```

Comment:
Why is `options` `var`? Or is this part of the "this PR does not clean up"-part?

Reply:
> Or is this part of the "this PR does not clean up"-part?

no longer :) thanks for the hint, i took the chance to tweak visibility and mutability